integration-tests/utils/Applications.ts

```diff
@@ -17,17 +17,27 @@ import { ComponentPage } from '../support/pages/ComponentsPage';
 import { CreateApplicationPage } from '../support/pages/CreateApplicationPage';
 import { ComponentsTabPage } from '../support/pages/tabs/ComponentsTabPage';
 import { OverviewTabPage } from '../support/pages/tabs/OverviewTabPage';
-import { PipelinerunsTabPage } from '../support/pages/tabs/PipelinerunsTabPage';
 import { Common } from './Common';
 
 export class Applications {
   static deleteApplication(applicationName: string) {
```

Comment:
I think `create-application-from-sample.spec` also uses the same method in test to delete the application.
I think it would be nice to have one test that deletes from UI in it() block.

Reply:
does it look better now? :)